### ISSUE TITLE
Should the reconstruction loss on the VAE Example be multiplied by the number of pixels in the image?

### ISSUE THREAD

**Issue by ronaldiscool** [pytorch/examples#817]
F.binary_cross_entropy takes the average of y_gt log (y_rec) + (1-y_gt) log (1-y_rec) across all pixels. However, shouldn't the correct loss for the VAE be the sum rather than the mean?
